flavors/squid_nlb_central/squidvm.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+SUB_FOLDER="/home/ubuntu/cloud-automation/"
+MAGIC_URL="http://169.254.169.254/latest/meta-data/"
+
+
+cd /home/ubuntu
+sudo apt-get update
+sudo apt-get install -y build-essential wget libssl-dev
+wget http://www.squid-cache.org/Versions/v4/squid-4.0.24.tar.xz
```

Comment:
Would you add some instructions on where to go to check for new versions, and a note on why we're running v4 before it goes GA, and that we'll switch to the `apt` package once v4 is available in the standard repos?

Reply:
done